src/components/ContentPreview/ContentPreview.js

```diff
@@ -911,9 +951,14 @@ class ContentPreview extends PureComponent<Props, State> {
                     )}
                     <div className='bcpr-body'>
                         <div className='bcpr-container' onMouseMove={this.onMouseMove} ref={this.containerRef}>
-                            <Measure bounds onResize={this.onResize}>
-                                {({ measureRef: previewRef }) => <div ref={previewRef} className='bcpr-content' />}
-                            </Measure>
+                            {!file ? (
+                                <PreviewLoading isError={!!isFileError} />
```

Comment:
Does this need the explicit `!!` cast or do we know it's a boolean?

Reply:
yes, as it could be undefined and Flow expects a boolean